web_translate_dialog/static/src/js/web_translate_dialog.js

```diff
@@ -225,5 +217,6 @@ common.AbstractField.include({
 return {
     translateDialog: translateDialog,
 };
+// odoo.define
+});
 
-}); // odoo.define
```

Comment:
I hate these kind of comments. Seems absurd even to have to see a diff on them. I'd be glad if you just drop it.